content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -377,70 +352,45 @@ Once the connection is established, Odoo can be synchronized with CodaBox.
          :menuselection:`Accounting --> Configuration --> Accounting: CodaBox`.
 
 .. note::
-    The connection between Odoo and CodaBox can be revoked by using the
-    :guilabel:`Revoke` button in the CodaBox settings, which will revoke
-    the link on Odoo's side only.
-
-    If you have multiple connections, you will still be able to use the
-    Access Token of one of them to easily reconnect Odoo and CodaBox for
-    another client.
-
-    However, if you have only one (remaining) connection, you will not be able
-    to use its Access Token to create new connections. In that case, you will
-    have to revoke the connection from the CodaBox platform too,
-    and then recreate a new one following the steps in the
-    :ref:`first connection <belgium/codabox-configuration-connection>` section.
+    The connection between Odoo and CodaBox can be revoked by going to
+    :menuselection:`Accounting --> Configuration --> Accounting: CodaBox --> Manage Connection`
+    and clicking on the :guilabel:`Revoke` button.
 
 Potential issues
```

Comment:
- I think the best solution for this section would be to use **bold** for the first line (the sentence(s) stating the potential issue), and maybe even a list (not sure about this one, up to you). That way you'd still be able to comply with the 100-char limit while keeping the section neat and clear. Don't forget to change the alignment of the "solution" lines :slightly_smiling_face: 
- I would also use guilabel where you're referring to the actual field labels (namely, Accounting Firm VAT and Company VAT), or take out the capital letters altogether if you're not.
- This is just a suggestion, but what about adding refs in the solution paragraphs so that people could click on the link and be taken directly to where the steps to perform are described? I don't think it's necessary for every solution, but it could be useful e.g., where you talk about revoking access or creating a new connection. What do you think?

Reply:
Done, it's indeed cleaner :)